python/nano/src/bigdl/nano/pytorch/encryption/torch/dispatcher.py

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# TODO: consider move this to nano's dispatcher
+
+_torch_patch = None
+is_torch_patched = False
+
+
+def patch_torch():
+    """
+    patch_torch is used to patch torch.save and torch.load methods to replace original ones.
+
+    Patched details include:
+
+    | 1. torch.save is renamed to torch.old_save
+    | 2. torch.load is renamed to torch.old_load
```

Comment:
since these 2 patching are kind of something we don't need our users to know, we may just skip them.

Reply:
Thanks, will fix